__tests__/Example-test.js

```diff
@@ -0,0 +1,3 @@
+it('works', () => {
+  expect(1).toBe(1);
+});
```

Comment:
What works? A little too vague.

Reply:
This is just to test that Jest works - it will be deleted as we add actual tests :)

Reply:
https://stackoverflow.com/questions/30571875/whats-the-difference-between-super-and-superprops-in-react-when-using-e